deps/sphinx-dependencies.sh

```diff
@@ -61,6 +61,7 @@ sudo pip install --upgrade requests
 
 # Install Sphinx (as root)
 sudo pip install Sphinx
+sudo pip install sphinx_rtd_theme
```

Comment:
The dependencies for software-factory, the other project which uses sphinx, is in that repo's Dockerfile:
https://github.com/Pelagicore/software-factory/blob/master/Dockerfile
I can agree that since more than one of our projects uses sphinx it looks nicer to install common dependencies in the cookbook, but if the arch is built by itself we will have issues, right?